### DISCUSSION TITLE
can't connect MQTT

### DISCUSSION THREAD

**Discussion by jerryaycock** [pixelit-project/PixelIt#88] [General]
Just finished project. Clock is running.
I can't get it to connect to my MQTT. I am running Home Assistant and have several 
components connecting to it.
I tried all 1883/tcp	
1883
Normal MQTT
1884/tcp	
1884
MQTT over WebSocket
8883/tcp	
8883
Normal MQTT with SSL
8884/tcp	
8884
MQTT over WebSocket with SSL
And nothing from the test area sends to the display

Thanks Jerry

**Comment by o0shojo0o** [comment]
Hi, 
I don't use MQTT myself, but I don't know of any problems from other users. 

The area of the WebUi does not use MQTT, but only websocket!  
If the test area does not work it is very unusual.

**Reply by jerryaycock** [reply]
Tomorrow, I will reflash and try again. Thanks

**Reply by jerryaycock** [reply]
Everything works great until I try to add mqtt. 
Node red in HA works great .. 
Thanks for the Great Project

**Comment by o0shojo0o** [comment]
@foorschtbar you also use MQTT, can you say something here?

**Reply by foorschtbar** [reply]
Hey. Yes, MQTT works fine in my setup. i never used the REST API.

I use a standalone MQTT server (Mosquitto). Are u sure that your Home Assistant MQTT Broker works?

**Reply by jerryaycock** [reply]
I am using Mosquito in Home Assistant. 
I can't connect.

**Reply by foorschtbar** [reply]
...and are u sure that Mosquito is set up correct? Try for example with MQTT.fx

**Reply by jerryaycock** [reply]
Mosqito  works great. I have all working without MQTT- I will carry on from here. Thanks for all help!
